[2024-03-29T08:42:18.758+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag_target.run_this manual__2024-03-29T08:42:17.491452+00:00 [queued]>
[2024-03-29T08:42:18.767+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag_target.run_this manual__2024-03-29T08:42:17.491452+00:00 [queued]>
[2024-03-29T08:42:18.767+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-03-29T08:42:18.780+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): run_this> on 2024-03-29 08:42:17.491452+00:00
[2024-03-29T08:42:18.786+0000] {standard_task_runner.py:60} INFO - Started process 2816 to run task
[2024-03-29T08:42:18.790+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'demo_dag_target', 'run_this', 'manual__2024-03-29T08:42:17.491452+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag_target.py', '--cfg-path', '/tmp/tmpm7sf5big']
[2024-03-29T08:42:18.792+0000] {standard_task_runner.py:88} INFO - Job 6: Subtask run_this
[2024-03-29T08:42:18.803+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-29T08:42:18.842+0000] {task_command.py:423} INFO - Running <TaskInstance: demo_dag_target.run_this manual__2024-03-29T08:42:17.491452+00:00 [running]> on host 4cab8a94f129
[2024-03-29T08:42:18.916+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='domwatcharin' AIRFLOW_CTX_DAG_ID='demo_dag_target' AIRFLOW_CTX_TASK_ID='run_this' AIRFLOW_CTX_EXECUTION_DATE='2024-03-29T08:42:17.491452+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-29T08:42:17.491452+00:00'
[2024-03-29T08:42:18.917+0000] {demo_dag_target.py:10} INFO - Remotely received value: Hello World
[2024-03-29T08:42:18.917+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-29T08:42:18.925+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=demo_dag_target, task_id=run_this, execution_date=20240329T084217, start_date=20240329T084218, end_date=20240329T084218
[2024-03-29T08:42:18.977+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-29T08:42:18.988+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
